### PR TITLE
Fix 'export()' args to include 'tinyxml2'

### PR DESCRIPTION
**Pull request by Gasparoken**
This is part of the issue aseprite/aseprite#2525

### CODE REVIEW

**Comment by dacap** [comment]
@Gasparoken I'll not be able to review this until next week, but just in case: did you try just cloning/forking the TinyEXIF project as it is? It looks like you've changed the project name (to lower case) when you forked it, but it might be better just to keep the original name (as it's quite hard to keep this kind of renaming when merging upstream changes).

**Comment by Gasparoken** [comment]
Yes, I can fork the TinyEXIF project as is.

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,18 +19,9 @@ set(GENERIC_LIB_SOVERSION "1")
...
-# By Default shared libray is being built
-# To build static libs also - Do cmake . -DBUILD_STATIC_LIBS:BOOL=ON
-# User can choose not to build shared library by using cmake -DBUILD_SHARED_LIBS:BOOL=OFF
-# To build only static libs use cmake . -DBUILD_SHARED_LIBS:BOOL=OFF -DBUILD_STATIC_LIBS:BOOL=ON
-# To build the demo binary, use cmake . -DBUILD_DEMO:BOOL=ON
-
-option(BUILD_SHARED_LIBS "build as shared library" ON)
-option(BUILD_STATIC_LIBS "build as static library" OFF)
-option(LINK_CRT_STATIC_LIBS "link CRT static library" OFF)
-option(BUILD_DEMO "build demo binary" ON)
```
Comment (dacap) [inline CMakeLists.txt:33]: This configuration must be specified from Aseprite repository from [third_party/CMakeLists.txt](https://github.com/aseprite/aseprite/blob/main/third_party/CMakeLists.txt), see how it's done with other libraries.

File: CMakeLists.txt [hunk 2]
```diff
@@ -92,7 +83,7 @@ if(BUILD_SHARED_LIBS)
 
 
 	if(DEFINED CMAKE_VERSION AND NOT "${CMAKE_VERSION}" VERSION_LESS "2.8.11")
-		target_include_directories(TinyEXIF PUBLIC 
+		target_include_directories(TinyEXIF PUBLIC
```
Comment (dacap) [inline CMakeLists.txt:86]: Avoid any extra unnecessary change in the file, we should have the minimal number of changes with upstream. Even if there are whitespaces.

File: CMakeLists.txt [hunk 3]
```diff
@@ -118,22 +109,22 @@ if(BUILD_SHARED_LIBS)
 			ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
 
-if(BUILD_STATIC_LIBS)
-	add_library(TinyEXIFstatic STATIC TinyEXIF.cpp TinyEXIF.h)
-	
-	target_link_libraries(TinyEXIFstatic tinyxml2::tinyxml2)
-	set_target_properties(TinyEXIFstatic PROPERTIES
+if(NOT BUILD_SHARED_LIBS)
+	add_library(TinyEXIF STATIC TinyEXIF.cpp TinyEXIF.h)
```
Comment (dacap) [inline CMakeLists.txt:113]: Are these changes necessary? could just not depend on `TinyEXIFstatic` directly?

File: CMakeLists.txt (hunk 2 above)
Comment (Gasparoken) [reply CMakeLists.txt:86]: My text editor is set to automatically remove unnecessary whitespace.
I really don't think it's convenient to have to -before the 'push'- deactivate this feature, add the empty spaces that the editor automatically deleted, do the 'push', and then reactivate the feature.

Comment (dacap) [reply CMakeLists.txt:86]: You can control what change you stage/include in the commit. I'll try to take the issue from here so we can include the submodule.

**Comment by dacap** [comment]
I've done some minor changes in https://github.com/aseprite/TinyEXIF/commit/2283d8094e42e6b6cef00d1d31ecadfe0d9e1f29 so we are able to use TinyEXIF from Aseprite without any other diff.
